docs/ml-features.md

```diff
@@ -1386,6 +1386,102 @@ print(output.select("features", "clicked").first())
 {% endhighlight %}
 </div>
 </div>
+## RFormula
```

Comment:
Insert an empty line to make it a section title. You can run `SKIP_API=1 jekyll build` under `spark/docs/` to build the user guide.